Style: modify form page UI, closes #81

KristelKolkanen opened the pull request.

Treimart reviewed (changes requested):
There is an error and the page cannot be opened because there is `<>` missing at the start of the `return` statement.

KristelKolkanen commented:
> There is an error and the page cannot be opened because there is `<>` missing at the start of the `return` statement.

fixed the error and changed "variant" `<Typography variant='h4'>Lisa uus kategooria</Typography>` in SaveNewCategory.tsx line 32

Treimart reviewed (approved):
Working great now!

I do think that textfields are a little too grouped together (would use bigger margins) but nothing too distracting 🙂
